### ISSUE TITLE
b2 build warning in Jamfile

### ISSUE THREAD

**Issue by jeking3** [boostorg/contract#11]
Probably been there a long time...

```
../libs/contract/test/Jamfile.v2:417: Unescaped special character in argument <define>BOOST_CONTRACT_ON_MISSING_CHECK_DECL="{ throw err(); }"
```

**Comment by jeking3** [comment]
Warning-free builds are better, though.  If it's a simple fix, please consider it.

**Comment by lcaminiti** [comment]
I will not generate a new release just for this, but I will consider to include this improvement in the next release when/if that is needed.

**Comment by lcaminiti** [comment]
I will fix this as suggested in pull request #12.

**Comment by lcaminiti** [comment]
fixed in develop branch (will merge in master for next boost release).
